assets/js/index.js

```diff
@@ -0,0 +1,9 @@
+// Instantiating
+var post = new Post();
+var blogs = new AppView({collection: post});
```

Comment:
You passed in a model as a collection... Also do this initialization as part of the `router.initialize` function so that the router is the central location for wiring your data to your views.